.github/workflows/test_names.txt

```diff
@@ -22,4 +22,17 @@ should change columns
 should work with enums (case 1)
 should work with enums (case 2)
 should work with enums with schemas
-should be able to remove a column with primaryKey
\ No newline at end of file
+should be able to remove a column with primaryKey
+should make the autoincremented values available on the returned instances
+should make the autoincremented values available on the returned instances with custom fields
+is possible to use functions as default values
+doesn't allow case-insensitive duplicated records using CITEXT
+allows the creation of a TSVECTOR field
+TSVECTOR only allow string
+doesn't allow duplicated records with unique function based indexes
+sets auto increment fields
+should return autoIncrement primary key \(create\)
+several concurrent calls
+should error correctly when defaults contain a unique key and the where clause is complex
+should work with multiple concurrent calls
+should not deadlock with concurrency duplicate entries and no outer transaction
```

Comment:
nit: small request, add EOL so we don't see a diff for the last line when we edit it.

Reply:
I think the new regex will fix it. Good suggestion.

Reply:
UPDATE: The regex did not work. The best way is to refactor the ignored tests to a programmatic usage.